library/waku_thread/inter_thread_communication/requests/peer_manager_request.nim

```diff
@@ -46,13 +48,15 @@ proc connectTo(node: WakuNode,
 
   return ok()
 
-method process*(self: PeerManagementRequest,
-                node: ptr WakuNode): Future[Result[string, string]] {.async.} =
+proc process*(self: ptr PeerManagementRequest,
+              node: ptr WakuNode): Future[Result[string, string]] {.async.} =
```

Comment:
this can be plain `WakuNode` because the function does not write to `node`

Reply:
Fixed. Thanks!